docs/visual-basic/programming-guide/language-features/declared-elements/access-levels.md

```diff
@@ -18,53 +19,73 @@ ms.assetid: 6e06c1ab-fd78-47f0-83a8-1152780b5e1a
   
 ## Protected Friend  
- The `Protected` and `Friend` keywords together in the declaration statement specify that the elements can be accessed either from derived classes or from within the same assembly, or both. The following code shows a sample `Protected Friend` declaration.  
+ The `Protected` and `Friend` keywords together in the declaration statement specify that the element can be accessed either from derived classes or from within the same assembly, or both. The following code shows a sample `Protected Friend` declaration.  
   
-```  
+```vb  
 Protected Friend stringForProjectAndHeirs As String  
 ```  
   
  You can use `Protected Friend` only at class level, and only when you declare a member of a class. This means you can declare a protected friend element in a class, but not at the level of a source file or namespace, or inside an interface, module, structure, or procedure.  
   
 ## Private  
- The [Private](../../../../visual-basic/language-reference/modifiers/private.md) keyword in the declaration statement specifies that the elements can be accessed only from within the same module, class, or structure. The following code shows a sample `Private` declaration.  
+ The [Private](../../../../visual-basic/language-reference/modifiers/private.md) keyword in the declaration statement specifies that the element can be accessed only from within the same module, class, or structure. The following code shows a sample `Private` declaration.  
   
-```  
+```vb  
 Private numberForMeOnly As Integer  
 ```  
   
  You can use `Private` only at module level. This means you can declare a private element inside a module, class, or structure, but not at the level of a source file or namespace, inside an interface, or in a procedure.  
   
  At the module level, the `Dim` statement without any access level keywords is equivalent to a `Private` declaration. However, you might want to use the `Private` keyword to make your code easier to read and interpret.  
-  
-## Access Modifiers  
+
+## Private Protected
+
+The [Private Protected](../../../language-reference/modifiers/private-protected.md) keyword in the declaration statement specifies that the element can be accessed only from within the same class, as well as from derived classes found in the same assembly as the containing class. The `Private Protected` access modifier is supported starting with Visual Basic 15.5.
+
+The following example shows a `Private Protected` declaration:
+
+```vb
+Private Protected internalValue As Integer
+```
+
+You can declare a `Private Protected` only inside of a class. You cannot declare it within an interface or structure, nor can you declare it at the level of a source file or namespace, inside an interface or a structure, or in a procedure.
+
+To use the `Private Protected` access modifier, you must add the following element to your Visual Basic project (*.vbproj) file:
```

Comment:
I'd add something about "15.5", or newer or "latest" to select the most recent version,  That way it's future proof.